code/modules/mob/living/living.dm

```diff
@@ -120,6 +120,13 @@ default behaviour is:
 			// VOREStation Edit - Begin
 			// Handle grabbing, stomping, and such of micros!
 			if(handle_micro_bump_other(tmob)) return
+			// Plow that nerd.
+			if(ishuman(tmob))
+				var/mob/living/carbon/human/H = tmob
+				if(H.species.lightweight == 1 && prob(50))
+					H.Weaken(20)
```

Comment:
T-that's a long knockdown time! But seeing as the cost is -4, I can support this

Also, if it doesn't already, may want to make it include a custom knockdown message. Otherwise, it'll just look like someone just rested for no reason.